app/src/components/search/collectionSearch.tsx

```diff
@@ -7,9 +7,20 @@ import {
 } from "@nypl/design-system-react-components";
 import DCSearchBar from "./dcSearchBar";
 import { useState } from "react";
+import type { SearchManager } from "@/src/utils/searchManager";
+import { usePathname, useRouter } from "next/navigation";
 
-export const CollectionSearch = () => {
+type CollectionSearchProps = {
+  searchManager: SearchManager;
+};
+
+export const CollectionSearch = ({ searchManager }: CollectionSearchProps) => {
   const [isExpanded, setIsExpanded] = useState(false);
+  const { push } = useRouter();
+  const pathname = usePathname();
+  const updateURL = async (queryString) => {
```

Comment:
You could use the parent's `updateURL` and pass it as a prop but I see that this doesn't need the `setIsLoaded(false);` call. Does it hurt to call it anyway since it's being set to false?